### PR TITLE
Added some documentation for encrypting with age 

### PR DESCRIPTION
**Pull request by simcax**
I have added some examples of having age recipients in the .sops.yaml as well as a little info about using it with updatekeys

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -204,6 +204,30 @@ identity will be tried in sequence until one is able to decrypt the data.
 
 Encrypting with SSH keys via age is not yet supported by sops.
 
+A list of age recipients can be added to the .sops.yaml:
```
Comment (felixfontein) [inline README.rst:207]:
```suggestion
A list of age recipients can be added to the ``.sops.yaml``:
```

File: README.rst [hunk 2]
```diff
@@ -204,6 +204,30 @@ identity will be tried in sequence until one is able to decrypt the data.
...
+.. code:: yaml
+
+    creation_rules:
+        - age: >-
+            age1s3cqcks5genc6ru8chl0hkkd04zmxvczsvdxq99ekffe4gmvjpzsedk23c,
+            age1qe5lxzzeppw5k79vxn3872272sgy224g2nzqlzy3uljs84say3yqgvd0sw
+
+It is also possible to use updatekeys, when adding or removing age recipients.
+
+I.e. :
```
Comment (felixfontein) [inline README.rst:218]:
```suggestion
It is also possible to use ``updatekeys``, when adding or removing age recipients. For example:
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! I've added two comments. Could you please rebase the result, and squash everything into one commit and make sure to sign off on it (for DCO)? Thanks.

**Comment by felixfontein** [comment]
Can you please squash the commits and sign-off on the resulting commit (see https://github.com/getsops/sops/pull/1001/checks?check_run_id=16863365696)?

**Comment by simcax** [comment]
> Can you please squash the commits and sign-off on the resulting commit (see https://github.com/getsops/sops/pull/1001/checks?check_run_id=16863365696)?

Sure thing, will do it tonight or tomorrow.

File: config/config.go [hunk 3]
```diff
@@ -8,7 +8,9 @@ import (
 	"io/ioutil"
 	"os"
 	"path"
+	"path/filepath"
 	"regexp"
+	"strings"
```
Comment (felixfontein) [inline config/config.go:13]: This PR now contains changes that should not be part of it. I guess something went wrong during rebasing?

Comment (simcax) [reply config/config.go:13]: Yes, I must have messed it up. Will give it a look again - sorry about the mess @felixfontein

**Comment by felixfontein** [comment]
Ping @simcax

**Pull request closed without merging**